operators/src/source/gdal_source/mod.rs

```diff
@@ -739,11 +739,13 @@ impl OperatorName for GdalSource {
 impl RasterOperator for GdalSource {
     async fn _initialize(
         self: Box<Self>,
+        path: WorkflowOperatorPath,
         context: &dyn crate::engine::ExecutionContext,
     ) -> Result<Box<dyn InitializedRasterOperator>> {
         let meta_data: GdalMetaData = context.meta_data(&self.params.data).await?;
 
         debug!("Initializing GdalSource for {:?}.", &self.params.data);
+        debug!("GdalSource prefix: {:?}", path);
```

Comment:
why not call it path?

Reply:
https://github.com/geo-engine/geoengine/pull/780/commits/eb4d17538a5d7b7395669e3d204ef04cf988e1d4